charactersheet/charactersheet/templates/dm/encounter_sections/map_section.tmpl.html

```diff
@@ -0,0 +1,181 @@
+    </div>
+      <div class="modal-body">
+        <form class="form-horizontal" onsubmit="return false;">
+          <div class="form-group">
+            <label class="col-sm-2 control-label">Name</label>
+            <div class="col-sm-10">
+              <input type="text"
+                     class="form-control"
+                     placeholder="King Arthur"
+                     data-bind="textInput: blankMap().name,
+                        hasFocus: firstElementInModalHasFocus">
+            </div>
+          </div>
+          <div class="form-group">
+            <label class="col-sm-2 control-label">Link</label>
+            <div class="col-sm-10">
+              <input type="text"
+                     class="form-control"
+                     placeholder=""
+                     data-bind="textInput: blankMap().imageUrl">
+            </div>
+          </div>
+          <div class="modal-footer">
+            <button type="button"
+                    class="btn btn-primary"
+                    data-bind='click: addMap'
+                    data-dismiss="modal">Add</button>
+          </div>
+        </form>
+      </div> <!-- Modal Body -->
+    </div> <!-- Modal Content -->
+  </div> <!-- Modal Dialog -->
+</div> <!-- Modal Fade -->
+
+<!-- ViewEdit Modal -->
+<div class="modal fade"
+     id="viewMap"
+     tabindex="-1"
+     role="dialog"
+     data-bind="modal: {
+      open: openModal,
+      onopen: modalFinishedOpening,
+      onclose: modalFinishedClosing
+    }">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button"
+                class="close"
+                data-dismiss="modal"
+                aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+        <h4 class="modal-title">Edit a Map</h4>
+      </div>
+      <div class="modal-body">
+        <!-- Begin Tabs -->
+        <ul class="nav nav-tabs tabs">
+          <li role="presentation" data-bind="click: selectPreviewTab, css: previewTabStatus">
+            <a href="#" aria-controls="itemsModalPreview" role="tab" data-toggle="tab">
+                <b>Preview</b>
+            </a>
+          </li>
+          <li role="presentation" data-bind="click: selectEditTab, css: editTabStatus">
+            <a href="#" aria-controls="itemsModalEdit" role="tab" data-toggle="tab">
+                <b>Edit</b>
+            </a>
+          </li>
+        </ul>
+        <div class="tab-content" data-bind="with: currentEditItem">
+          <div role="tabpanel" data-bind="css: $parent.previewTabStatus" class="tab-pane">
+            <div class="h3">
+              <span data-bind="text: name"></span>
+            </div>
+            <hr />
+            <div class="row row-padded">
+              <div class="col-xs-12 col-padded text-center">
+                <img data-bind="attr: { src: imageUrl }" class="image-rounded" width="100%" />
```

Comment:
I can't find this CSS class: `image-rounded`.
Also, if an image is too tall, it will extend the modal down. Do we want this behavior or do we want to implement a `max-height`?
Also, if an image is too small, it will stretch it to fit the 100% width of the modal. Is this okay?